benchopt/config.py

```diff
@@ -261,6 +293,29 @@ def get_setting(name, config_file=None, benchmark_name=None,
     return value
 
 
+def get_data_path(key: str = None, config_file=None, benchmark_name=None,
+                  default_config=None):
```

Comment:
We should only have parameter `key`, the other should be inferred from the benchmark.
To get the current benchmark, you can use the novel `get_running_benchmark` function.